python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -418,6 +418,76 @@ def test_actor_method_bind_same_input_attr(ray_start_regular):
     compiled_dag.teardown()
 
 
+def test_actor_method_bind_diff_input_attr_1(ray_start_regular):
+    actor = Actor.remote(0)
+    with InputNode() as inp:
+        # Two class methods are bound to two different input
+        # attribute nodes.
+        output1 = actor.inc.bind(inp[0])
+        output2 = actor.inc.bind(inp[1])
+        dag = MultiOutputNode([output1, output2])
+    compiled_dag = dag.experimental_compile()
+    ref = compiled_dag.execute(0, 1)
+    assert ray.get(ref) == [0, 1]
+
+    ref = compiled_dag.execute(1, 2)
```

Comment:
can you also add a test for dict input? 

```
ref = compiled_dag.execute({"a:" 1, "b": 2})
```

Reply:
just one test is fine

Reply:
Updated: https://github.com/ray-project/ray/pull/47489/commits/c3628810770a5f35874bd864728543e47c1a3c12